composer.json

```diff
@@ -66,9 +66,6 @@
 	"extra": {
 		"branch-alias": {
 			"dev-master": "2.7.x-dev"
-		},
-		"symfony": {
-			"require": "~4.4|~5.4|^6.3"
```

Comment:
IMHO removing this is fine since there should be no problem installing `3.4` packages, in this case, this was covering the fact that there is this call:

https://github.com/craue/CraueConfigBundle/blob/d71792fbf7d108c5783ab5b3e30e41d19c5962db/CacheAdapter/DoctrineCacheBundleAdapter.php#L40

And `CacheProvider::saveMultiple()` [was added in `1.6.0`](https://github.com/doctrine/cache/pull/117), so we should have had a conflict with `doctrine/cache` < 1.6.

I guess this `extra.symfony.require` only applies to this package, so removing it would help to find issues that users can have.

Reply:
Okay, tests pass with the conflict constraint added. 👍🏻